Jenkinsfile

```diff
@@ -61,6 +61,28 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             currentBuild.description = "⚡ ${newBuildID}"
         }
 
+        stage('Test') {
+            // clone & build kola
+            utils.shwrap("""
+            git clone https://github.com/arithx/mantle
+            cd mantle
+            git checkout fcos_ci
+            ./build kola kolet
+            """)
+            
+            // determine the image & run kola
+            utils.shwrap("""
+            latest_build=\$(readlink builds/latest)
+            qcow=\$(ls builds/"\${latest_build}"/*-"\${latest_build}"-qemu.qcow2)
+            mantle/bin/kola -p unprivileged-qemu --qemu-image "\${qemu}" -b fcos run | tee
```

Comment:
IMO something like this should be [coreos-assembler kola](https://github.com/coreos/coreos-assembler/pull/85).